lxd/container_lxc.go

```diff
@@ -6796,9 +6808,10 @@ func (c *containerLXC) setNetworkLimits(name string, m types.Device) error {
 
 	// Look for the host side interface name
 	veth := c.getHostInterface(m["name"])
-
 	if veth == "" {
-		return fmt.Errorf("LXC doesn't know about this device and the host_name property isn't set, can't find host side veth name")
+		return fmt.Errorf(`LXC doesn't know about this device and the ` +
+			`host_name property isn't set, can't find host side ` +
+			`veth name`)
```

Comment:
I'm not convinced by this type of wrapping, it makes grepping for errors much harder than it should be...